### ISSUE TITLE
res.locals from within filter/function?

### ISSUE THREAD

**Issue by ericclemmons** [twigjs/twig.js#57]
Hi!  Not sure _if_ this is an issue (yet :D), but several filters we use in PHP (and would like to use within Express as well) behave slightly different based on [res.locals](http://expressjs.com/api.html#res.locals) as well as [app.locals](http://expressjs.com/api.html#app.locals).

For example, the former may automatically add the right class to a navigation element depending on the current request's route, while the latter may automatically add a debug flag when the `app.settings.env === 'dev'` or something.

In short, I'm wondering **can twig.js functions and/or filters have access to the `app` or local view variables, and if so, how?**

Thanks!
